NeXus/NXxpcs.nxdl.xml

```diff
@@ -76,146 +76,48 @@
-        </field>
-        <field name="g2_stderr" type="NX_NUMBER" units="NX_ANY">
-          <doc>TODO:</doc>
-        </field>
-        <field name="tau" type="NX_INT" units="NX_ANY">
-          <doc>
-          delay time, multitau
-
-          t_el, this is integer delta frame units, not time elapsed units
-          </doc>
-        </field>
-      </group>
-      <group type="NXdata" name="twotime">
-        <!-- TODO: -->
-        <field name="C2" type="NX_NUMBER" units="NX_ANY">
-          <doc>two-time correlation</doc>
-        </field>
-        <field name="g2_twotime" type="NX_NUMBER" units="NX_ANY">
-          <doc>g2full, sum across the diagonals of C2</doc>
-        </field>
-        <field name="g2_partials_twotime" type="NX_NUMBER" units="NX_ANY">
-          <doc>g2 by subset of frames</doc>
-        </field>
-      </group>
-    </group>
-
-    <group type="NXcanSAS" name="SAXS_1D">
-      <!--
-        https://manual.nexusformat.org/classes/applications/NXcanSAS.html
-
-        TODO: Should this be type="NXsubentry" with definition field of NXcanSAS?
-
-        Structure from our initial example file:
-
-            SAXS_1D:NXsubentry
-              @NX_class = NXsubentry
-              @canSAS_class = SASentry
-              @default = data
-              definition:NX_CHAR = NXcanSAS
-              instrument -> /entry/instrument
-              run -> /entry/title
-              sample -> /entry/sample
-              title:NX_CHAR = static 1D SAXS from XPCS data
-              data:NXdata
-                @I_axes = Q
-                @NX_class = NXdata
-                @Q_indices = 0
-                @canSAS_class = SASdata
-                @signal = I
-                I:NX_FLOAT32[270] = [0.27429348, 0.34260944, 0.2445109, '...', 0.0057929275]
-                  @units = arbitrary
-                  @units_details = Photon/Pixel/Frame
-                Q:NX_FLOAT64[270] = [0.0018368, 0.00186642, 0.001891, '...', 0.063412]
-                  @units = 1/angstrom ->
-
-      -->
-      <doc>Additions to the NXcanSAS definition.</doc>
-      <group type="NXdata" name="data">
-        <field name="I_partial" type="NX_NUMBER">
-          <doc>
-            I(Q) for stability plot.
+      <field name="frame_sum" type="NX_NUMBER" units="NX_ANY">
+        <doc>average intensity v. time</doc>
+      </field>
+      <field name="g2" type="NX_NUMBER" units="NX_ANY">
+        <doc>TODO:</doc>
+      </field>
+      <field name="g2_stderr" type="NX_NUMBER" units="NX_ANY">
+        <doc>TODO:</doc>
+      </field>
+      <field name="tau" type="NX_INT" units="NX_ANY">
+        <doc>
+        delay time, multitau
 
-            Q: Should units match I?
-          </doc>
-        </field>
-        <!-- defined in NXcanSAS -->
-        <field name="I" type="NX_NUMBER" />
-        <field name="Q" type="NX_NUMBER" units="NX_PER_LENGTH" />
-      </group>
+        t_el, this is integer delta frame units, not time elapsed units
```

Comment:
Was that the final result from that email discussion?

Reply:
Good point.  Time to add the results from the email discussions now.

Reply:
I still do not understand the result of the discussion.

Reply:
This quantity is the period of the time clock for XPCS measurements (usually a very small time number).